Add Egress analytics dashboard artifacts and edit cache dashboard to use response event stream (#8751)
wso2/docs-apim: en/docs/api-analytics/on-prem/elk-installation-guide.md

5. Download the artifact file from below.<br />
   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Artifacts v1 - [here]({{base_path}}/assets/img/analytics/cloud/old_export.ndjson)<br />
   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Artifacts v2 - [here]({{base_path}}/assets/img/analytics/cloud/export.ndjson)
   <br />
   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Artifacts v3 - [here]({{base_path}}/assets/img/analytics/cloud/export_v3.ndjson)

!!! info

>     If you have AI APIs as well, use Artifacts v3 file.

Reviewer: Shall we reword like so:
```suggestion
    If you have any AI APIs, make sure to use Artifacts v3 file.
```
Also, remove the extra new line that follows this info box
